docker-compose.yml

```diff
@@ -1,13 +1,14 @@
-version: '3'
+version: '3.4'
 services:
-  tests: &tests-config
+  release:
     build: .
     image: keboola/manage-api-tests
-    tty: true
-    stdin_open: true
+    command: composer ci
+
+  dev: &dev-config
```

Comment:
Dev se jmenuje dev, protože je to image který používáš při vývoji. Prostě chceš vyvíjet? Použij dev.

Sice na něm lokálně pouštíš testy, ale klidně v něm můžeš pustit i phpcs.